public/js/post_action.js

```diff
@@ -1,67 +0,0 @@
-// Force the Cover card to follow scroll
```

Comment:
This file is now empty, I guess we can remove it?

Reply:
I had done that, but probably a rebase or a merge messed with that. Removed it now :+1: